Source/Analytics/Web/src/assets/main.scss

```diff
@@ -1,3 +1,98 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400&display=swap');
+
+html {
+    font-size: 16px;
+    font-family: "Lato", "Helvetica Neue", sans-serif;
+}
+
 body {
-  margin: 0;
+    margin: 0;
+}
+
+$rc-red: #ca0000;
+$light-grey: #f0f0f0;
+$medium-grey: #cfcdcc;
+
+$screen-xs: 480px;
```

Comment:
these aren't needed?

Reply:
I don't think all of these variables are needed, but they are used to design for different screen sizes. I think they can be written in a smarter way though, this is very verbose.